src/features/contactverzoek/service.ts

```diff
@@ -246,6 +268,22 @@ export function fetchVragenSets(url: string) {
     });
 }
 
+function mapToClientContactVerzoekVragenSets(
+  serverDataArray: ServerContactVerzoekVragenSet[],
+): ContactVerzoekVragenSet[] {
+  return serverDataArray.map((serverData) => {
+    const parsedQuestions = mapSchemaToVragen(
```

Comment:
ik vraag me af wat de toegevoegde waarde is van het mappen naar aparte interfaces, waarom niet 1 interface met een property questiontype om verderop onderscheidt te kunnen maken tussen de verschillende soorten